[2025-07-18T09:19:11.109+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-18T09:19:11.114+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data manual__2025-07-18T02:16:47.597572+00:00 [queued]>
[2025-07-18T09:19:11.117+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data manual__2025-07-18T02:16:47.597572+00:00 [queued]>
[2025-07-18T09:19:11.117+0700] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2025-07-18T09:19:11.131+0700] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_load_weather_data> on 2025-07-18 02:16:47.597572+00:00
[2025-07-18T09:19:11.137+0700] {standard_task_runner.py:63} INFO - Started process 6026 to run task
[2025-07-18T09:19:11.138+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag', 'transform_load_weather_data', 'manual__2025-07-18T02:16:47.597572+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpku06g8m5']
[2025-07-18T09:19:11.139+0700] {standard_task_runner.py:91} INFO - Job 78: Subtask transform_load_weather_data
[2025-07-18T09:19:11.172+0700] {task_command.py:426} INFO - Running <TaskInstance: weather_dag.transform_load_weather_data manual__2025-07-18T02:16:47.597572+00:00 [running]> on host MSI.
[2025-07-18T09:19:11.223+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='menglimchea@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='transform_load_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-18T02:16:47.597572+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-18T02:16:47.597572+00:00'
[2025-07-18T09:19:11.224+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-18T09:19:11.465+0700] {credentials.py:670} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2025-07-18T09:19:12.869+0700] {logging_mixin.py:188} INFO - âœ… Saved to: s3://menglim/current_weather_data_phnompenh_18072025091911.csv
[2025-07-18T09:19:12.870+0700] {python.py:237} INFO - Done. Returned value was: None
[2025-07-18T09:19:12.870+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-18T09:19:12.875+0700] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=transform_load_weather_data, run_id=manual__2025-07-18T02:16:47.597572+00:00, execution_date=20250718T021647, start_date=20250718T021911, end_date=20250718T021912
[2025-07-18T09:19:12.916+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-18T09:19:12.925+0700] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-18T09:19:12.925+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
